README.md

```diff
@@ -156,6 +156,31 @@ New `wasm-code` binary file will be generated in the `runtime` folder.
 
 You should have [Docker](https://docker.io) installed.
 
+### Docker image from Docker hub (option 1)
+
+We have a Docker Hub repository where we host the latest image of the whole project. This is the easiest and fastest way to run the Substrate node with Assemblyscript Runtime.
+
+First, pull the image from the Docker Hub. 
+``` 
+docker pull limechain/as-substrate
+```
+
+Then run the executable:
+
+```
+docker run -p 9933:9933 --name node-runtime limechain/as-substrate \
+    --port 30333 \
```

Comment:
I think that the dockerfile should expose this port as-well otherwise you won't be able to run a network of nodes using the docker images.